### PR TITLE
SpotBugs integrated into build

### PR DESCRIPTION
**Pull request by Bitked**
Addresses issue #104. Integrating SpotBugs into the build using the existing buiild,grade file.

Additionally added some spaces to separate code blocks for easier readability.

### CODE REVIEW

**Comment by Bitked** [comment]
Core plugin was added.

Additional information on adding community plugins are found on the issue #119 link to gradle.org if needed later by developers.

**Comment by jody** [comment]
Please suggest a reviewer.

**Comment by Bitked** [comment]
@henrykgray would you please reveiw this request

File: build.gradle [hunk 1]
```diff
@@ -41,7 +43,9 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.1.0'
     // These dependancies are for backwards compatibility with JUnit3/4
     testCompileOnly 'junit:junit:4.12'
-    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.1.0'}
+    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.1.0'
```
Comment (henrykgray) [inline build.gradle]: Change is irrelevant to the user story. Maybe a bug report could be created to resolve.

**Comment by jody** [comment]
"spotbugs" tasks should be run, and not interfere with the running of other automated assessments, following the invocation of the `gradlew check` command.

**Pull request closed without merging**